runtimes/spiritnet/Cargo.toml

```diff
@@ -179,6 +185,8 @@ std = [
   "pallet-authorship/std",
   "pallet-balances/std",
   "pallet-collective/std",
+  "pallet-dip-provider/std",
+  "pallet-deposit-storage/std",
```

Comment:
Can you put them in the right order? It makes it easier to see if we miss some features.

Reply:
Like, where's the `std` for `pallet-configuration` and `kilt-runtime-api-dip-provider`?

Reply:
Changed order in: [8810616](https://github.com/KILTprotocol/kilt-node/pull/616/commits/8810616671f2aa36e1f36563e1443e05516fb510)

The only differences between the spiritnet cargo.toml vs. the peregrine cargo.toml should come from the sudo and configuration pallet.